cmd/kube-scheduler/app/server.go

```diff
@@ -323,10 +319,19 @@ func Setup(ctx context.Context, opts *options.Options, outOfTreeRegistryOptions
 		scheduler.WithPodInitialBackoffSeconds(cc.ComponentConfig.PodInitialBackoffSeconds),
 		scheduler.WithExtenders(cc.ComponentConfig.Extenders...),
 		scheduler.WithParallelism(cc.ComponentConfig.Parallelism),
+		scheduler.WithBuildFrameworkCapturer(func(profile kubeschedulerconfig.KubeSchedulerProfile) {
```

Comment:
Add a comment to clarify why is this done.

Reply:
That's a "what", not a "why" 😉 

I would say something like: "Profiles are processed during Framework instantiation to set default plugins and configurations. Capturing them for logging".